public/static/docs/command-reference/list.md

```diff
@@ -0,0 +1,130 @@
+# list
+
+List <abbr>repository</abbr> contents, including files and directories tracked
+by DVC (<abbr>data artifacts</abbr>) and by Git.
+
+## Synopsis
+
+```usage
+usage: dvc list [-h] [-q | -v] [-R] [--outs-only] [--rev [REV]] url [target]
+
+positional arguments:
+  url              Location of DVC repository to list.
+  target           Path to directory within the repository to list contents for.
+```
+
+## Description
+
+List files, dirs and <abbr>data artifacts</abbr> for the pointed url. The output
+is sorted lexicographically.
```

Comment:
Can we provide a brief motivation? Like regular ls or Github web is not enough because it does not take into account DVC managed files and directories. And in a lot of cases we would like to explore this before we do import/get for example (so that we able to see the path actual model is actually stored) ... etc